src/CabanaPD_Particles.hpp

```diff
@@ -164,6 +165,21 @@ class Particles<MemorySpace, PMB, Dimension>
         createParticles( exec_space );
     }
 
+    // Constructor which initializes particles on regular grid with
+    // customization.
+    template <class ExecSpace, class UserFunctor>
+    Particles( const ExecSpace& exec_space, std::array<double, dim> low_corner,
+               std::array<double, dim> high_corner,
+               const std::array<int, dim> num_cells, const int max_halo_width,
+               UserFunctor user_create )
+        : halo_width( max_halo_width )
+        , _plist_x( "positions" )
+        , _plist_f( "forces" )
```

Comment:
@streeve : Why do we add here "_plist_f( "forces" )"? Is that for initial forces?

Should we add other initial particles properties such as initial displacements too?